config/prod/rna-seq-reprocessing-instance-v001.yaml

```diff
@@ -0,0 +1,32 @@
+# Provision an EC2 instance
+template_path: toil-instance.yaml
+stack_name: rna-seq-reprocessing-instance-v001
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "tess.thyer@sagebase.org"
+
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "toil"
+  AMIId: "ami-032e0d8fe5cfa8c7f"  # toil-ubuntu ami
+  RoleStackName: "rna-seq-reprocessing-instance-role-v001"
+
+  RsaKey: !ssm /toil-cluster-infra/SshKey
+  PublicRsaKey: !ssm /toil-cluster-infra/PublicSshKey
```

Comment:
@zaro0508 I make these secure string params in sandbox, but will need your assistance to get them into scicomp.